main.py

```diff
@@ -42,6 +43,10 @@ def h_init(args):
 	with open(confpath, "w") as f:
 		f.write(json.dumps(config, sort_keys=True, indent=4))
 
+	if not os.path.exists(permpath):
+		with open(permpath, "w") as f:
+			f.write(json.dumps({}, sort_keys=True, indent=4))
```

Comment:
Minor nit - this should actually be `[]` here instead of `{}` for the empty list.

Permission lists are an empty list, not an empty object (which I may have accidentally said in the past). This isn't actually a problem worth redoing the whole PR or whatever for, but should probably be fixed in the next convenient PR.

Note: I don't think this will actually break anything (loading the empty object will still produce an empty permissions list), but it is technically inconsistent with what should be there.